content/400-reference/200-api-reference/050-prisma-client-reference.mdx

```diff
@@ -565,6 +565,49 @@ const result = await prisma.user.findUnique({
 })
 ```
 
+##### Get the <inlinecode>Product</inlinecode> record with a given <inlinecode>name</inlinecode>, and containing a <inlinecode>Photo</inlinecode> composite type with a given <inlinecode>url</inlinecode>
```

Comment:
```suggestion
##### Get the <inlinecode>Product</inlinecode> record with a specific <inlinecode>name</inlinecode>, and containing a <inlinecode>Photo</inlinecode> composite type with a specific <inlinecode>url</inlinecode>
```

I'm suggesting this because "given name" has a meaning - I think the heading reads better with an alternative wording